### PR TITLE
add citation

### PR DESCRIPTION
**Pull request by Roger-luo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,18 @@ pkg> add Bloqade#master
 
 You can join QuEra's Slack workspace with this [link](https://join.slack.com/t/querapublic/shared_invite/zt-1d5jjy2kl-_BxvXJQ4_xs6ZoUclQOTJg). Join the `#bloqade` channel to discuss anything related to Bloqade.
 
+## Citation
+
+We are still working on a paper for this package, but if you use Bloqade in your research, please cite the following:
```
Comment (Wang-Shengtao) [inline README.md:49]:
```suggestion
If you use Bloqade in your research, please cite the following:
```

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# - family-names: "John"
+#   given-names: "Doe"
+
+title: "Bloqade.jl: Package for the quantum computation and quantum simulation based on the neutral-atom architecture."
+version: "v1"
+license: "Apache License 2.0"
+# doi: "10.22331/q-2020-10-11-341"
+date-released: 2023-01-09
+url: "https://github.com/QuEraComputing/Bloqade.jl"
+# NOTE: add the reference paper here in the future
```
Comment (Wang-Shengtao) [inline CITATION.cff:14]: Remove the commented lines below?

Comment (Wang-Shengtao) [reply CITATION.cff:14]: I meant lines 14-34 here. Looks like it's copied from Yao. Should we remove those lines?

File: README.md [hunk 3]
```diff
@@ -44,6 +44,18 @@ pkg> add Bloqade#master
 
 You can join QuEra's Slack workspace with this [link](https://join.slack.com/t/querapublic/shared_invite/zt-1d5jjy2kl-_BxvXJQ4_xs6ZoUclQOTJg). Join the `#bloqade` channel to discuss anything related to Bloqade.
 
+## Citation
+
+If you use Bloqade in your research, please cite the following:
```
Comment (weinbe58) [inline README.md:49]:
```suggestion
If you use Bloqade for a publication, we would kindly ask you to cite our work using the following bibtex citation:
```

**Review by weinbe58 — COMMENTED** [review]
changing the language a bit to make it a bit more friendly.

**Review by Wang-Shengtao — APPROVED** [review]

**Pull request merged**